docs/data-management/metrics/simple-metric.md

```diff
@@ -148,6 +148,28 @@ For example, consider a podcast app with a fact that logs **podcast listen event
 * _Episodes listened per user_, to measure overall content consumption. This can be captured correctly with a `Count` metric. Since repeated listens to the same episode are rare, using `Count Distinct` on episode id would give similar results but incur higher warehouse costs.
 :::
 
+#### Last Value
+
+Last Value computes the most recent value of a fact. It can be interpreted as averages across entities:
+
+$\frac{\text{SUM of most recent fact value}}{\text{Number of unique entities assigned}}$
```

Comment:
> It can be interpreted as averages across entities:
Helpful to call this out like we do for `SUM`, or no?

Reply:
Yes good to have this defined.

Can we clarify it's the most recent value of a numeric fact?

Reply:
Fixed!